website/docs/faqs/Accounts/find-user-id.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Where can I find my user id?
+description: "Instructions on where to find your user id"
+sidebar_label: 'Where can I find my user id'
+id: find-user-id
+---
+
+You may need to find your user id to rotate your API token, and so on. Refer to the following steps to find your user id in dbt Cloud:
```

Comment:
```suggestion
Refer to the following steps to find your user ID in dbt Cloud:
```

Reply:
Unless we want to expound on "and so on" the first part of this feels a little excessive.